Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -111,7 +111,7 @@ void FullscreenManager::requestFullscreenForElement(Ref<Element>&& element, Full
             return completionHandler(Exception { ExceptionCode::TypeError, message });
         ERROR_LOG(identifier, message);
         if (emitErrorEvent == EmitErrorEvent::Yes) {
-            m_fullscreenErrorEventTargetQueue.append(WTFMove(element));
+            m_pendingEvents.append(std::pair { EventType::Error, WTFMove(element) });
```

Comment:
I had to specify `std::pair`, otherwise it wouldn't let me `WTFMove` a `Ref` into `GCReachableRef`. Happy to take any suggestions for improvements if anyone has any.